crates/matrix-sdk/src/sliding_sync/room.rs

```diff
@@ -125,6 +125,12 @@ impl SlidingSyncRoom {
         self.inner.client.clone()
     }
 
+    /// Find the latest event in this room
+    #[cfg(feature = "experimental-sliding-sync")]
```

Comment:
I reckon the entire module is behind the `experimental-sliding-sync` feature, we can remove it here :-).